draft-ietf-jsonpath-base.md

```diff
@@ -1479,9 +1483,9 @@ parameter of declared type `LogicalType`, with the equivalent conversion rule:
   * If the nodelist contains one or more nodes, the conversion result is `LogicalTrue`.
   * If the nodelist is empty, the conversion result is `LogicalFalse`.
 
-Extraction of a value from a nodelist can be performed in several
-ways, so an implicit conversion from `NodesType` to `ValueType`
-may be surprising and has therefore not been defined.
```

Comment:
I've attempted a rewording of this in https://github.com/ietf-wg-jsonpath/draft-ietf-jsonpath-base/pull/456.